packages/shared/src/components/modals/ResponsiveModal.tsx

```diff
@@ -10,16 +10,26 @@ import styles from './ResponsiveModal.module.css';
 
 export const responsiveModalBreakpoint = breakpoint;
 
+interface ResponsiveModalProps extends ModalProps {
+  noPadding?: boolean;
+}
+
 export function ResponsiveModal({
   className,
+  noPadding,
```

Comment:
Instead of providing a `noPadding` property consider using the already existing `style` property to style the content element